include/mbedtls/mbedtls_config.h

```diff
@@ -1118,7 +1118,7 @@
  *           MBEDTLS_ECP_DP_SECP256R1_ENABLED
  *
  * \warning If SHA-256 is provided only by a PSA driver, you must call
- * psa_crypto_init() before the first hanshake (even if
+ * psa_crypto_init() before the first handshake (even if
```

Comment:
The typo `hanshake` still exists in `tests/ssl-opt.sh` (in function names)

Reply:
Thanks. I have fixed the function names too.